orc8r/tools/ansible/roles/pkgrepo/tasks/main.yml

```diff
@@ -14,6 +14,27 @@
   become: yes
   ignore_errors: yes
 
+- name: Wait for APT Lock
+  shell: |
+    while sudo fuser /var/lib/dpkg/lock >/dev/null 2>&1 ; do
```

Comment:
Maybe add some logging? Just to understand where the code might be waiting.

Reply:
The current output already is
```
 TASK [pkgrepo : Wait for APT Lock] *********************************************
Thursday 23 June 2022  09:21:38 +0000 (0:00:00.562)       0:00:05.431 ********* 
changed: [magma] => {"changed": true, "cmd": "while sudo fuser /var/lib/dpkg/lock >/dev/null 2>&1 ; do\n  sleep 1\ndone\nwhile sudo fuser /var/lib/apt/lists/lock >/dev/null 2>&1 ; do\n  sleep 1\ndone\nif [ -f /var/log/unattended-upgrades/unattended-upgrades.log ]; then\n  while sudo fuser /var/log/unattended-upgrades/unattended-upgrades.log >/dev/null 2>&1 ; do\n    sleep 1\n  done\nfi\n", "delta": "0:00:00.135144", "end": "2022-06-23 09:21:38.599628", "msg": "", "rc": 0, "start": "2022-06-23 09:21:38.464484", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
```
Admittedly, I think this is only printed when the step is finished, but then again you don't usually look at the logs while this is running.